tests/test_api.py

```diff
@@ -3,15 +3,15 @@
     require_api_key,
     require_connect,
 )
-from rsconnect.api import RSConnect, RSConnectExecutor, _to_server_check_list
+from rsconnect.api import RSConnectClient, RSConnectExecutor, _to_server_check_list
```

Comment:
Looks like this change breaks the `rsconnect-jupyter` plugin. `rsconnect-jupyter` tries to import `RSConnect` so we hit this error when installing/enabling `rsconnect-jupyter`:

```
ImportError: cannot import name 'RSConnect' from 'rsconnect.api'
```

Reply:
For now I think we can just set `RSConnect = RSConnectClient` in api to fix this.

Reply:
Confirmed this has been fixed.  Still validating remaining workflows, will update when testing is complete.